packages/roosterjs-content-model-types/lib/parameter/TrustedHTMLHandler.ts

```diff
@@ -1,4 +1,16 @@
 /**
  * A handler type to convert HTML string to a trust HTML string
  */
-export type TrustedHTMLHandler = (html: string) => string;
+export type LegacyTrustedHTMLHandler = (html: string) => string;
```

Comment:
Let's mark this interface as "@deprecated" in comment. So next time we do major version bump, we will remove it.